topic_name : /scan
queue_size : 50
p_gain : 0.5
